### DISCUSSION TITLE
Browse->Filesystem view

### DISCUSSION THREAD

**Discussion by vernonjvs** [jcorporation/myMPD#637] [Q&A] [answered]
With Development version 8.2.0, I no longer see the Add View To Home Screen icon for the Browse->Filesystem view. Is this intentional? Please note that the Browse->Playlists and Browse->Database pages still show the  Add View To Home Screen icon.

Thanks.

**Comment by jcorporation** [comment]
This is intentionally. You can add directories through the contextmenu or the add button to the homescreen.

**Reply by vernonjvs** [reply]
How is eliminating the ability to add the  Browse->Filesystem view to the Home screen an improvement? 

Thanks.

**Reply by jcorporation** [reply]
There are many more actions available for directories as for views. I can readd the option for the root to be more consistent.

Thanks for testing this devel release.

**Reply by vernonjvs** [reply]
Yes, please readd  Browse->Filesystem view Home screen option. In addition to consistency, this is the default ympd view which is what I have been using and it is also the fastest view. My Home screen has this Queue, Filesystem and Playlists view which is what I primarily use.

Thanks

**Comment by jcorporation** [comment, marked as answer]
Commit e98465fcbba2004ddb2419322edaa52ff2694203 readded the add to homescreen to the filesystem root dir.

**Reply by vernonjvs** [reply]
Thanks very much!
